x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/alerts_table_state.tsx

```diff
@@ -308,10 +302,18 @@ const AlertsTableStateWithQueryProvider = ({
     updatedAt,
   ]);
 
+  const CasesContext = casesService?.ui.getCasesContext();
+  const casesPermissions = casesService?.helpers.canUseCases();
+  const isCasesContextAvailable = casesService && CasesContext;
+
   const tableProps: AlertsTableProps = useMemo(
     () => ({
       alertsTableConfiguration,
-      casesData: { cases: cases ?? new Map(), isLoading: isLoadingCases },
+      cases: {
+        data: cases ?? new Map(),
+        isLoading: isLoadingCases,
+        showBulkActions: Boolean(isCasesContextAvailable && alertsTableConfiguration.cases),
```

Comment:
Shouldn't we check the `casesPermissions` here? I see we are rendering the "add to Case" bulk actions in the alert table, even when the user can not read/write Cases.

![capabilities_bug](https://user-images.githubusercontent.com/17747913/226340706-430226be-61bc-42cf-af5e-463db2e06b02.png)

When these actions are clicked the Case flyout/modal opens, and it throws an error when submitting because the user does not have privileges to write Cases. 

In the previous implementation (main) the Cases actions are not rendered if the user does not have permission to write Cases. We should probably keep the same behavior.

Reply:
Thank you for finding this @semd! I fixed it. Let me know if you need me to address anything else.

Reply:
We found the bug happens when the Security Cases capability is disabled, but any other (O11y or Management) Cases is enabled. Thanks @cnasikas for addressing it.